consensus/parlia/parlia.go

```diff
@@ -221,11 +221,12 @@ type Parlia struct {
 
 	lock sync.RWMutex // Protects the signer fields
 
-	ethAPI                     *ethapi.PublicBlockChainAPI
-	VotePool                   consensus.VotePool
-	validatorSetABIBeforeLuban abi.ABI
-	validatorSetABI            abi.ABI
-	slashABI                   abi.ABI
+	ethAPI                      *ethapi.PublicBlockChainAPI
+	VotePool                    consensus.VotePool
+	validatorSetABIBeforeLuban  abi.ABI
+	validatorSetABIBeforeFusion abi.ABI
```

Comment:
It is never used. I believe it is the ABI after the fusion, isn't it?

Reply:
Deleted